### DISCUSSION TITLE
Job log showing "app-disabled", info panel shows 404s when trying to retrieve logs

### DISCUSSION THREAD

**Discussion by leahecole** [renovatebot/renovate#19690] [Request Help]
Hi renovate folks, I'm trying to resolve a warning on a repo I help maintain and wanted to verify that the fix we put in worked. However, only 1 job log in the past three days shows up as "done", the rest show up as "app-disabled" even though we haven't changed our configuration in months. Any ideas for how to troubleshoot this?

The "done" job log does render properly when you click on it - the rest throw 404s. 

![image](https://user-images.githubusercontent.com/6719667/210845556-4ef5f6b1-ca77-4700-b68e-f3388598c9f7.png)

**Comment by rarkins** [comment]
Hi Leah, it seems that you're using the Forking Renovate app for that repo, which unfortunately has not supported log viewing before now. I'm actually a little surprised that you see the list of jobs at all because you would logged in with the regular Renovate app to see that screen.

The good news is that we have a new UI for Forking Renovate in testing, which you can get access to: https://developer.mend.io

**Reply by leahecole** [reply]
Oh awesome, good to know! Thanks for the info! 🙂
